airflow/migrations/utils.py

```diff
@@ -20,7 +20,7 @@
 from contextlib import contextmanager
 
 
-def get_mssql_table_constraints(conn, table_name):
+def get_mssql_table_constraints(conn, table_name) -> defaultdict(list):
```

Comment:
```suggestion
def get_mssql_table_constraints(conn, table_name) -> dict[str, dict[str, list[str]]]:
```

I don’t think you can _call_ `defaultdict` here… (same for other calls below). We can do `defaultdict[...]` but it’s not necessary, a `dict` is good enough.

Reply:
🤦,I did some of the refactoring in a semi automatic way, and missed that spot in the review, thanks

Reply:
@uranusjr, just fixed that, ty :)